onboarding/source/controller.c

```diff
@@ -94,7 +109,7 @@ static void ledTimerCallback(TimerHandle_t xTimer) {
 static void lightTimerCallback(TimerHandle_t xTimer) {
     /* USER CODE BEGIN */
     // Send light event to light service queue
-
+    sendToLightServiceQueue(MEASURE_LIGHT);
```

Comment:
The `sendToLightServiceQueue` function expects a pointer. You need to pass in a pointer to a variable that has a value of `MEASURE_LIGHT`.